curl.yaml

```diff
@@ -58,12 +74,26 @@ subpackages:
     description: "documentation for curl"
     pipeline:
       - uses: split/manpages
-  - name: "libcurl4"
-    description: "curl library"
+  - name: "libcurl-openssl4"
+    description: "curl library (openssl backend)"
+    dependencies:
+      provides:
+        - libcurl4=7.87.1
```

Comment:
why this version differs from main package?

Reply:
We want to provide some extra weight to these packages verses the original libcurl4.